.github/workflows/testing-test-build-app.yml

```
@@ -0,0 +1,35 @@
+name: "Testing Test Build App"
+
+on:
+    merge_group:
+    pull_request:
+        types: [ opened, synchronize ] # Don't rerun on `edited` to save time
+    workflow_call:
+
+jobs:
+  testing-build-app:
+    runs-on: buildjet-8vcpu-ubuntu-2204
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        submodules: recursive # Needed in order to fetch Kalium sources for building
+        fetch-depth: 0
+    - name: Set up JDK 17
+      uses: buildjet/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: gradle
+    - name: Build Staging Compat
+      env:
+        GITHUB_USER: ${{ github.actor }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        ./gradlew assembleStagingCompat -p ./ --no-daemon
+    - name: Upload APK staging compat
+      if: success()
+      uses: ./.github/workflows/upload-aws-s3.yml
```

Can you try to use, just for testing purposes, I have the feeling this is a temporary problem, and might be solved after merge to the default branch (`develop`)

```suggestion
      uses: ./.github/workflows/upload-aws-s3.yml@feat/github-actions-upload-to-s3
```